src/main/java/com/shapesecurity/salvation/directives/SourceListDirective.java

```diff
@@ -39,6 +36,10 @@ public boolean matchesNonce(@Nonnull String nonce) {
             .anyMatch(x -> ((MatchesNonce) x).matchesNonce(nonce));
     }
 
+    public boolean containsUnsafeInlineDisabler() {
```

Comment:
I would rather separately expose `containsHashSource` and `containsNonceSource`.